Lesson4/src/com/home/model/fabric/Fabric.java

```diff
@@ -1,8 +1,106 @@
 package com.home.model.fabric;
 
-import com.home.model.fabric.data.MyData;
 
-public abstract class Fabric<T> {
+import java.util.List;
+
+public interface Fabric<T> {
+
+    T getSomeObject(List<String> initData);
+}
+
+// Old realization
```

Comment:
get rid of this if you don't need it here, please

Reply:
removed